WEB-INF/lib/ttTimeHelper.class.php

```diff
@@ -843,28 +843,6 @@ static function getAllRecords($user_id) {
 
     return $result;
   }
-
-  //return range of dates
-  static function getAllDateRecords($from_date,$to_date) {
```

Comment:
Why is this deleted?

Reply:
i made a separate file for it and put it sir.
that function was made by me.